kolibri/utils/options.py

```diff
@@ -352,3 +352,24 @@ def update_options_file(section, key, value, KOLIBRI_HOME, ini_filename="options
             file=conf.filename
         )
     )
+
+
+def generate_empy_options_file(options_path, options_data):
+
+    exclude_key = ["PICKLE_PROTOCOL"]
+
+    # Generate an options.ini file inside the KOLIBRI_HOME as default placeholder config
+    file = open(options_path, "w")
```

Comment:
Do you mind changing this to a `with` statement so in case any exception is raised in the for loop, we can make sure the file handle gets closed? Thank you!

Reply:
```suggestion
    with open(options_path, "w") as file:
```